src/main/kotlin/main.kt

```diff
@@ -0,0 +1,33 @@
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import org.radarcns.gateway.Config
+import org.radarcns.gateway.GrizzlyServer
+import java.io.File
+import java.io.IOException
+import kotlin.system.exitProcess
+
+fun main(args: Array<String>) {
+    val config = if (args.size == 1) {
+            try {
+                val mapper = ObjectMapper(YAMLFactory())
+                mapper.readValue(File(args[0]), Config::class.java)
+            } catch (ex: IOException) {
+                println("Usage: radar-gateway [config.yml]")
+                println("Failed to read config file ${args[0]}: ${ex.message}")
+                exitProcess(1)
+            }
+        } else Config()
```

Comment:
Why do we have them in a separate `Config` class  instead of using the default config file gateway.yml (since it already exists)?

Reply:
it serves as a default if no configuration file is provided